sql/information_schema/information_schema.go

```diff
@@ -1533,17 +1531,21 @@ func NewInformationSchemaDatabase() Database {
 				schema: columnStatisticsSchema,
 				reader: columnStatisticsRowIter,
 			},
-
+			ColumnsTableName: &ColumnsTable{
+				name:    ColumnsTableName,
+				schema:  columnsSchema,
+				rowIter: columnsRowIter,
+			},
 			ColumnsExtensionsTableName: &informationSchemaTable{
 				name:   ColumnsExtensionsTableName,
 				schema: columnsExtensionsSchema,
 				reader: emptyRowIter,
 			},
-			ConnectionControlFailedLoginAttemptsTableName: &informationSchemaTable{
-				name:   ConnectionControlFailedLoginAttemptsTableName,
-				schema: connectionControlFailedLoginAttemptsSchema,
-				reader: emptyRowIter,
-			},
+			//ConnectionControlFailedLoginAttemptsTableName: &informationSchemaTable{
+			//	name:   ConnectionControlFailedLoginAttemptsTableName,
+			//	schema: connectionControlFailedLoginAttemptsSchema,
+			//	reader: emptyRowIter,
+			//},
```

Comment:
Was this one of the tables you mentioned was documented, but not actually present? If so, I think we can go ahead and pull this out instead of leaving it commented out.

Reply:
Yep, got it.